transit_odp/organisation/tests/csv/test_service_codes.py

```diff
@@ -121,92 +138,308 @@ def test_registered_service_code():
+
+    assert csv_output["row3"][0] == '"No"'
+    assert csv_output["row3"][1] == '"Unpublished"'
+    assert csv_output["row3"][2] == '"Registered"'
+    assert csv_output["row3"][3] == '"Out of Scope"'
+    assert csv_output["row3"][4] == '""'
+    assert csv_output["row3"][5] == '"Not Stale"'
+    assert csv_output["row3"][14] == '""'
+    assert csv_output["row3"][15] == '"True"'
+    assert csv_output["row3"][16] == '"2023-05-12"'
+
+    assert csv_output["row4"][0] == '"Yes"'
+    assert csv_output["row4"][1] == '"Published"'
+    assert csv_output["row4"][2] == '"Registered"'
+    assert csv_output["row4"][3] == '"In Scope"'
+    assert csv_output["row4"][4] == '"In Season"'
+    assert csv_output["row4"][5] == '"Stale - OTC Variation"'
+    assert csv_output["row4"][14] == '"2023-06-24"'
+    assert csv_output["row4"][15] == '"False"'
+    assert csv_output["row4"][16] == '"2023-02-10"'
+
+    assert csv_output["row5"][0] == '"Yes"'
+    assert csv_output["row5"][1] == '"Published"'
+    assert csv_output["row5"][2] == '"Registered"'
+    assert csv_output["row5"][3] == '"In Scope"'
+    assert csv_output["row5"][5] == '"Stale - End date passed"'
+    assert csv_output["row5"][14] == '"2024-06-23"'
+    assert csv_output["row5"][15] == '"False"'
+    assert csv_output["row5"][16] == '"2022-05-12"'
+
+    assert csv_output["row6"][0] == '"Yes"'
+    assert csv_output["row6"][1] == '"Published"'
+    assert csv_output["row6"][2] == '"Registered"'
+    assert csv_output["row6"][3] == '"In Scope"'
+    assert csv_output["row6"][5] == '"Stale - 12 months old"'
+    assert csv_output["row6"][14] == '"2023-01-24"'
+    assert csv_output["row6"][15] == '"False"'
+    assert csv_output["row6"][16] == '"2020-12-13"'
+
+    assert csv_output["row7"][0] == '"Yes"'
+    assert csv_output["row7"][1] == '"Published"'
+    assert csv_output["row7"][2] == '"Registered"'
+    assert csv_output["row7"][3] == '"In Scope"'
+    assert csv_output["row7"][5] == '"Stale - 12 months old"'
+    assert csv_output["row7"][14] == '"2023-01-24"'
+    assert csv_output["row7"][15] == '"False"'
+    assert csv_output["row7"][16] == '"2020-12-13"'
+
+    assert csv_output["row8"][0] == '"No"'
+    assert csv_output["row8"][1] == '"Published"'
+    assert csv_output["row8"][2] == '"Registered"'
+    assert csv_output["row8"][3] == '"In Scope"'
+    assert csv_output["row8"][5] == '"Not Stale"'
+    assert csv_output["row8"][14] == '"2024-02-24"'
+    assert csv_output["row8"][15] == '"False"'
+    assert csv_output["row8"][16] == '"2020-12-13"'
+
+    assert csv_output["row9"][0] == '"No"'
+    assert csv_output["row9"][1] == '"Published"'
+    assert csv_output["row9"][2] == '"Unregistered"'
+    assert csv_output["row9"][3] == '"In Scope"'
+    assert csv_output["row9"][4] == '""'
+    assert csv_output["row9"][5] == '"Not Stale"'
+    assert csv_output["row9"][14] == '"2024-02-24"'
+    assert csv_output["row9"][15] == '""'
+    assert csv_output["row9"][16] == '""'
```

Comment:
Can you add a test for "Stale - OTC Variation" please? It's the only one missing!